js/test/test.js

```diff
@@ -148,24 +148,19 @@ function exceptionMessage (exc) {
 // ----------------------------------------------------------------------------
 
 async function testMethod (methodName, exchange, ... args) {
-    let extraMessage = '';
-    if (!(methodName in exchange.has)) {
-        extraMessage = '[skip] missing definition in exchange';
-    } else if (exchange.has[methodName] === undefined) {
-        extraMessage = '[skip] undefined definition in exchange';
-    } else if (exchange.has[methodName] === false) {
-        extraMessage = '[pass] exchange does not support the endpoint';
+    let skipMessage = undefined;
+    if (!(methodName in exchange.has) || !exchange.has[methodName]) {
+        skipMessage = 'not supported';
```

Comment:
well, i dont think that is better. I think the detailed messaging was more correct.

Reply:
@ttodua I understand but that logic was a bit convoluted, and besides that, it does not add any useful information. It makes no difference if we're missing it because it's undefined here or there we can quickly check that, and having a lot of different "error" messages pollutes the output too much